creature/Player/player.cpp

```diff
@@ -42,4 +42,12 @@ void Player::action(sf::Event& event, float time, const Field& game_field) {
         else
             Action::stop(this);
     }
+}
+
+void Player::show_player(sf::RenderWindow& window) {
+    window.draw(_sprite);
+    if (get_armor().get_body() != nullptr)
+        window.draw(get_armor().get_body()->get_sprite());
+    if (get_armor().get_helmet() != nullptr)
+        window.draw(get_armor().get_helmet()->get_sprite());
 }
```

Comment:
The same function in enemy. Added it to creature and remove from player and enemy.

Reply:
Or you can add this function to Drawer, looks like this class can implement drawing logic of all objects.